HousingSearchListener/serverless.yml

```diff
@@ -26,6 +26,8 @@ functions:
       TenureApiToken: ${ssm:/housing-tl/${self:provider.stage}/tenure-api-token}
       AccountApiUrl: ${ssm:/housing-finance/${self:provider.stage}/account-api-url}
       AccountApiToken: ${ssm:/housing-tl/${self:provider.stage}/account-api-token}
+      FinancialTransactionApiUrl: ${ssm:/housing-finance/${self:provider.stage}/financial-transaction-api-url}
+      FinancialTransactionApiToken: ${ssm:/housing-tl/${self:provider.stage}/financial-transaction-api-token}
```

Comment:
the parameter name should start with `housing-finance `

Reply:
Maybe @LBHSPreston created it?

Reply:
I think so

Reply:
@LBHSPreston when you get a chance can you please confirm with the financial transactions api token parameter name should be, I think it should be `housing-finance/….` but want to make sure

Reply:
@humulla  I have modified the SSM parameter name, please have a look and approve the PR.